packages/main/src/AINotice.ts

```diff
@@ -0,0 +1,175 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import type { ResizeObserverCallback } from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import i18n from "@ui5/webcomponents-base/dist/decorators/i18n.js";
+import AINoticeTemplate from "./generated/templates/AINoticeTemplate.lit.js";
+import AINoticeCss from "./generated/themes/AINotice.css.js";
+import { AINOTICE_BUTTON_TOOLTIP } from "./generated/i18n/i18n-defaults.js";
+
+import Link from "./Link.js";
+import Button from "./Button.js";
+import Popover from "./Popover.js";
```

Comment:
you don't depend on the Popover - it's not used in the template. In the class it's only used as a type, so import it as a type - `import type Popover...`